tests/Integration/Behaviour/Features/Scenario/Product/bulk_duplicate_product.feature

```diff
@@ -0,0 +1,358 @@
+      | tax rules group | US-AL Rate (4%) |
+      | on_sale         | true            |
+      | wholesale_price | 70              |
+      | unit_price      | 500             |
+      | unity           | bag of ten      |
+    And I update product product1 SEO information with following values:
+      | meta_title[en-US]       | SUNGLASSES meta title |
+      | meta_description[en-US] | Its so smart          |
+      | meta_description[fr-FR] | lel joke              |
+      | link_rewrite[en-US]     | smart-sunglasses      |
+      | link_rewrite[fr-FR]     | lunettes-de-soleil    |
+      | redirect_type           | 301-product           |
+      | redirect_target         | product2              |
+    And I update product product1 shipping information with following values:
+      | width                                   | 10.5                 |
+      | height                                  | 6                    |
+      | depth                                   | 7                    |
+      | weight                                  | 0.5                  |
+      | additional_shipping_cost                | 12                   |
+      | delivery time notes type                | specific             |
+      | delivery time in stock notes[en-US]     | product in stock     |
+      | delivery time in stock notes[fr-FR]     | en stock             |
+      | delivery time out of stock notes[en-US] | product out of stock |
+      | delivery time out of stock notes[fr-FR] | En rupture de stock  |
+      | carriers                                | [carrier1,carrier2]  |
+    When I set product product1 suppliers:
+      | reference         | supplier reference | product supplier reference     | currency | price tax excluded |
+      | product1supplier1 | supplier1          | my first supplier for product1 | USD      | 10                 |
+    And I set following related products to product product1:
+      | product2 |
+    And I update product product1 with following customization fields:
+      | reference    | type | name[en-US]               | name[fr-FR]                         | is required |
+      | customField1 | text | text on top of left lense | texte en haut de la lentille gauche | true        |
+    And I add new attachment "att1" with following properties:
+      | description[en-US] | puffin photo nr1 |
+      | description[fr-FR] | macareux         |
+      | name[en-US]        | puffin           |
+      | name[fr-FR]        | macareux         |
+      | file_name          | app_icon.png     |
+    When I associate product product1 with following attachments: "[att1]"
+    And I enable product "product1"
+    When I update product product1 SEO information with following values:
+      | redirect_type   | 301-product |
+      | redirect_target | product2    |
+    And product product1 should have following seo options:
+      | redirect_type   | 301-product |
+      | redirect_target | product2    |
+
```

Comment:
The extra line should be removed, you can add a comment to indicate it's another product being created